src/alan_opcode_reference.md

```diff
@@ -0,0 +1,860 @@
+## Alan Opcode Reference
+
+The [Alan Standard Library](./standard_library.md), [AMM](./amm_language_reference.md), and [AGA](./aga_language_reference.md) formats all make extensive use of opcodes defined within the runtime to do useful work.
```

Comment:
It doesn't have to be in this PR, but we should have `./standard_library.md` link to https://github.com/alantech/alan/tree/main/std

Reply:
Should be easy.